source/common/runtime/runtime_features.cc

```diff
@@ -83,6 +83,7 @@ constexpr const char* runtime_features[] = {
     "envoy.reloadable_features.new_tcp_connection_pool",
     "envoy.reloadable_features.no_chunked_encoding_header_for_304",
     "envoy.reloadable_features.preserve_downstream_scheme",
+    "envoy.reloadable_features.rbac_policy_upstream_address",
```

Comment:
We shouldn't use runtime for this. You should have an actual config field for DFP that turns on this functionality, document it, and then refer to it in the matcher documentation.